.github/workflows/clang.yaml

```diff
@@ -159,7 +159,7 @@ jobs:
 
       - name: Merge Test Results
         run: |
-          junitparser merge --glob artifacts/**/twister.xml junit.xml
+          junitparser merge --glob "artifacts/**/twister.xml" junit.xml
```

Comment:
```suggestion
          junitparser merge --glob "artifacts/*/twister.xml" "artifacts/*/*/twister.xml" junit.xml
```

Reply:
oh I guess I should test with multiple parallel jobs too - you might very well be right, given [this](https://github.com/weiwei/junitparser/blob/689f9e1f1d9c4ca22ad62a07b5c24ca3718152cf/junitparser/cli.py#L86C29-L86C34) doesn't have `recursive = True`)

Reply:
yeah well I tested it locally